.github/workflows/build_frontend.yaml

```diff
@@ -3,7 +3,6 @@ name: Frontend
 on:
   pull_request:
     branches:
```

Comment:
Does this obsolete this workflow and `.github/actions/build-frontend` (used below)

Reply:
Currently, it does not. This only changes `pull_request`-based workflows (except `production`) as a starter. Push-based actions will still trigger the original flows as well as PRs for `production`. This can be applied wider after an observation period if dev-ops desires.